src/app/components/table_details/table_details.html

```diff
@@ -20,7 +20,7 @@ <h6>Details</h6>
                             <div class="detail-body">
                                 <dl class="detail" ng-repeat="column in row">
                                     <div ng-repeat="(k, v) in column">
-                                        <dt class="detail-label">{{ k }}</dt>
+                                        <dt data-toggle="tooltip" title="{{ k }}" class="detail-label">{{ k }}</dt>
```

Comment:
So this displays the same content in a tooltip, right? I don't think we should merge it, as a future feature to display some different content will necessarily change this line.
It will also be easier to handle communication with upstream and repo maintenance if the entire feature is implemented in the same PR. WDYT?

Reply:
I am not sure what you mean? This will not reach upstream, as this builds on a feature that was rejected upstream. The one feature to add additional rows of information with `meta_docs`. 

We do have to think about what we actually want to show in the tooltip, though! Something @sercancicek and me will have to figure out, how we can get an additional description in here.

Reply:
Ah I see, I thought the tooltip was for any `meta` information.

> We do have to think about what we actually want to show in the tooltip, though! Something @sercancicek and me will have to figure out, how we can get an additional description in here.

Got it. Sounds like this should be done before adding the tooltip then, because duplicating the content (as is currently done) does not bring anything new. Right?

Reply:
Yes we will have to work on that before we can merge this one indeed. Then we can adapt this PR!